### PR TITLE
Change columns for natsort on shared studies table

### LINKED ISSUE #894: js error loading the studies page
Getting this error when trying to open the study list: error

My guess is something to do with one of the studies not having certain value. We can't update main site until this is solved.

### PR DESCRIPTION
**Pull request by adamrp**
Fix #894

### CODE REVIEW

File: qiita_pet/templates/private_studies.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
         });
         $('#shared-studies-table').dataTable({
             order: [[ 0, "asc" ]],
-            columnDefs: [{type:'natural', targets:[1,2,3,7]}]
+            columnDefs: [{type:'natural', targets:[2,3,4]}]
```
Comment (antgonza) [inline qiita_pet/templates/private_studies.html]: Could you add a comment about which column each number represents?

Comment (adamrp) [inline qiita_pet/templates/private_studies.html]: It's just all the columns that don't have HTML in them. The HTML throws off the natsort plugin.

**Comment by antgonza** [comment]
:+1:

**Comment by ElDeveloper** [comment]
This works fine on my system, though I don't have a way to reproduce the problem.

**Comment by ElDeveloper** [comment]
:+1:

**Pull request merged**